deepmd/__init__.py

```diff
@@ -1,19 +1,12 @@
 """Root of the deepmd package, exposes all public classes and submodules."""
 
-from . import descriptor
-from . import fit
-from . import loss
-from . import utils
-from . import cluster
 import deepmd.utils.network as network
-from .infer.deep_eval import DeepEval
-from .infer.deep_pot import DeepPot
-from .infer.deep_dipole import DeepDipole
-from .infer.deep_polar import DeepPolar
-from .infer.deep_polar import DeepGlobalPolar
-from .infer.deep_wfc import DeepWFC
-from .infer.data_modifier import DipoleChargeModifier
+
+from . import cluster, descriptor, fit, loss, utils
 from .env import set_mkl
+from .infer import DeepPotential
+from .infer.data_modifier import DipoleChargeModifier
+from .infer.deep_eval import DeepEval
```

Comment:
I am not sure if it is a good idea to provide DeepEval, because it is not supposed to be called by the users.

Reply:
Yes you are right, that was left there by accident